src/app/pages/person-details/person-details.component.html

```diff
@@ -0,0 +1,52 @@
+<ng-container *ngIf="{ state: (isLoading$ | async) === false } as isLoading">
+  <ng-container *ngIf="details$ | async as details; else preloader">
+    <ng-container *ngIf="isLoading.state; else preloader">
+      <mdb-detailed-page-hero
+        [isPersonPage]="true"
+        [title]="details.name"
+        [img]="details.photo">
+        <ng-container description>
```

Comment:
What the purpose of this ng-container?

Reply:
CSS grid Layout for `detailed page-hero` component bulid that way, that we need to separate `description` and `other content` to properly position them on small resolution screens. As for person detailed page, we also need to separate  `known for` block from `other content`, in addition to `description`.
Other option i see to achieve needed behaviour it layout would be to add two conditional renderings(based on screen resolution) for description in two different places inside layout: one for desktop and second for mobile, but i think it's bad soution to rerender same component responsively.